docs/src/pages/api-reference/next.mdx

```diff
@@ -0,0 +1,27 @@
+## createNextRouteHandler
+
+```tsx
+import { createNextRouteHandler } from "uploadthing/next";
+
+import { ourFileRouter } from "./uploadthing.ts";
+
+export const { GET, POST } = createNextRouteHandler({
+  router: ourFileRouter,
+  config: {
```

Comment:
we have a `callbackUrl` option too: https://github.com/pingdotgg/uploadthing/blob/9b1e2b722f5a80d5bb6205491e753998e4420d6b/packages/uploadthing/src/internal/handler.ts#L130C3-L130C26